ddp-workspace/projects/ddp-dsm-ui/src/app/form-data/form-data.component.ts

```diff
@@ -18,9 +19,23 @@ export class FormDataComponent {
   @Output() patchData = new EventEmitter();
 
   currentPatchField: string;
+  TEXT_AREA_DEFAULT_SIZE = 50000;
+  TEXT_DEFAULT_SIZE = 200;
+  dynamicMaxLength = 100;
+  textareaField = new FormControl();
+
+  ngOnInit(): void {
+    if (this.fieldSetting.details && this.fieldSetting.details['size'] > 0) {
```

Comment:
**#Optional**
You could simplify like this: 
`if(this.fieldSetting?.['size'] > 0)`